src/user-managerment/module/individual/pages/IndividualCreatePin.js

```diff
@@ -15,7 +17,27 @@ const IndividualCreatePin = () => {
 
     // DATA INITIALIZATION
     const navigate = useNavigate()
-    const handleSubmit = ()=>{
+    const handleSubmit = (e)=>{
+        e.preventDefault()
+        // axios.post(`${BASE_URL}/api/User/AddPin`, pin)
+        // .then((response) => {
+        //     setIsComplete(response.data.message)
+        //     console.log(response.data.message)
+        // })
+        .catch((error) => {
+            alert(error)
+        })
+
+        // useEffect(()=>{
+        //     if(isSuccessful!==true && !(isEmpty(data))) {
+        //         toast.error(data.message)
+        //     } else if(isSuccessful===true && !(isEmpty(data))) {
```

Comment:
why was this commented out, @Ogooluwakiitan @michael7666? Is there an issue

Reply:
No, there isn't......I believe she cited that template for the "AddPin' API consumption.